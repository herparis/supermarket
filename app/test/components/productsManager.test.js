import React from 'react'
import { shallow } from 'enzyme'

import ProductsManager from '../../src/components/productsManager/productsManager'

function checkRender(wrapper) {
  expect(wrapper.isEmptyRender()).toBe(false)
  expect(wrapper).toMatchSnapshot()
}

describe('ProductsManager components rendeding', () => {
  const products = []
  const productsManager = shallow(<ProductsManager products={products} />)
  it('productsManager renders', () => {
    checkRender(productsManager)
  })
})