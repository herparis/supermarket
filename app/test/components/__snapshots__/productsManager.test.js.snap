// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductsManager components rendeding productsManager renders 1`] = `ShallowWrapper {}`;
